### DISCUSSION TITLE
🤯🎉 Special thanks to @kaalibro for fixing numerous UI issues in #458.

### DISCUSSION THREAD

**Discussion by canisminor1990** [lobehub/sd-webui-lobe-theme#460] [Announcements]
> [!IMPORTANT]
> @kaalibro Your contribution is greatly appreciated. 💞💞💞

#### 🔀 变更说明 | Description of Change

Sorry for suggesting so many changes in one commit, but each of them individually are pretty minor. They do not make major changes to the UI. Rather, they correct some shortcomings and make small improvements.

**UI:**
- Lightbox Modal redesigned, added as a new component 'lightboxModal'.
- Added more SVG icons with the ability to specify their size (necessary for Lightbox).
- Extensions styles added: ADetailer, ControlNet, Aspect Ratio selector, Inpaint Anything, Model Toolkit.
- Tools buttons now have proper width and do not take up extra space.
- Generation infotext simplified layout to improve readability.
- Extra Networks: removed unnecessary margin for tabnav items; background blur for card actions; fix for names word break.
- Metadata info: max width limited to 80% for better readability and to fix inability to close popup in some cases; font set to monospace.
- Theme Settings window: description line-height 1.3 for better readability.
- Quicksettings: items aligned more neatly; removed unnecessary 8px gap with the main working area.
- Styles editor popup: fix Styles dropdown label layout.
- PNG Info: full height image display.
- Global popup and Lightbox modal overlay unified look.
- txt2img, img2img settings controls and accordions layout fix.
- Token counters style slightly polished.
- Button transition set to '0.12s cubic-bezier'.
- Different colors for 'Skip' and 'Interrupt' buttons.
- New hover effect for generated images gallery item.
- Gallery min height increased to 470px.
- 'ETA' value placed to start of Progress bar.
- Increased WebUI Settings tab-nav width.

**Code:**
- Hardcoded buttons and font sizes replaced with vars for easier editing in 'tokens.ts' (if needed).
- Minor styles refactoring. A couple of unused classes removed.

#### 📝 补充信息 | Additional Information

Some examples:

![quicksettings-prompt](https://github.com/lobehub/sd-webui-lobe-theme/assets/44464226/d523c587-f3f1-47d6-a4ea-55a0256d3dc8)
![img-settings-gallery-infotext](https://github.com/lobehub/sd-webui-lobe-theme/assets/44464226/c09410d6-202f-440e-ae5c-440822f1417f)
![lightbox](https://github.com/lobehub/sd-webui-lobe-theme/assets/44464226/e558604b-dbd0-4aa7-b396-6896de914397)
![png-info](https://github.com/lobehub/sd-webui-lobe-theme/assets/44464226/de0294d6-37bf-430b-a717-2e8e98533f3d)
![extra-networks](https://github.com/lobehub/sd-webui-lobe-theme/assets/44464226/f9b29d83-75e6-46ed-9a00-bf9149f75adc)
![metadata-info](https://github.com/lobehub/sd-webui-lobe-theme/assets/44464226/34cb996a-2b44-4cd3-a03e-32be9f0ec979)
![styles-editor](https://github.com/lobehub/sd-webui-lobe-theme/assets/44464226/be1c9402-d4ba-4550-b7bf-e29fb4209d27)
![settings](https://github.com/lobehub/sd-webui-lobe-theme/assets/44464226/b930483a-4563-4690-a7c3-d92fb8cb7330)
![extra-svg-icons](https://github.com/lobehub/sd-webui-lobe-theme/assets/44464226/bd4257ba-730f-433a-bbf5-70deb66e4b83)

**Comment by SalehA-55** [comment]
Incredible work. Highly appreciated, all of you.

**Comment by bropines** [comment]
I just think that it’s better to make commits and releases of ordinary changes, and bring major updates directly here. This makes it easier to keep track of the project.
